### ISSUE TITLE
Jim is away for a week =)

### ISSUE THREAD

**Issue by jimsalterjrs** [jimsalterjrs/sanoid#813]
Hi contributors and other folks!

We're pretty overdue for a great big merge party soon, but I'll be out of town for the next week (8th-15th). Anybody who's got a PR in (and @phreaker0 ), can we maybe set up a merge day sometime the week of the 16th-23rd? We'll want PR authors on hand to rebase patches as needed; sometimes accepting one PR invalidates another until rebase.

Thanks!

**Comment by phreaker0** [comment]
hi @jimsalterjrs, the following PR are reviewed/tested and can be merged right away if you like:

#523 
#622 
#808 
#809

**Comment by phreaker0** [comment]
should be safe too: #746

**Comment by HankB** [comment]
If one of these PRs involves the `Usage:` produced when `sanoid -?` is executed, can the defaults be updated?

```text
    Assumes --cron --verbose if no other arguments (other than configdir)
    are specified
```

I just put 2.1.0 on a new machine and confirm that `sudo sanoid` creates no snapshots (or output) whereas `sudo sanoid --cron` does.

Should I submit a PR?

Thanks!

**Comment by jimsalterjrs** [comment]
#809 and #622 need to be rebased. The others merged okay. Haven't done any testing yet.

**Comment by phreaker0** [comment]
@jimsalterjrs rebased #809, will rebase #622 after merge of the former because this will trigger an conflict.

**Comment by jimsalterjrs** [comment]
@phreaker0 #809 merged.

**Comment by phreaker0** [comment]
@jimsalterjrs thanks, I rebased  #622

**Comment by jimsalterjrs** [comment]
#622 Merged.

**Comment by sarcasticadmin** [comment]
Thank you for merging all of these PRs @phreaker0 @jimsalterjrs 

Will there be an upcoming tag cut as well?

**Comment by msladek** [comment]
https://github.com/jimsalterjrs/sanoid/pull/625 has also just been rebased by @0xFelix the author. any chance of a merge @phreaker0 ?
without it sanoid still can't replicate from bookmarks (if there are any common snapshots).

**Comment by pgassmann** [comment]
@jimsalterjrs are you also planning to create a new release? the last one on github was 2 years ago, on April 1, 2021.
We are hitting a bug that was fixed more than a year ago. #734
How safe is it to use the script from the master branch?

**Comment by jimsalterjrs** [comment]
> How safe is it to use the script from the master branch?

Generally speaking, tends to be safe as kittens. @phreaker0 and I are the only ones with commit/merge privileges, and we have the explicit goal of keeping master as safe as we can between releases. Occasionally, one of us has merged a PR with a bug we didn't catch first... But I explicitly do all of my (pretty frequent) personal sanoid installations from master, so I generally catch any significant bugs in master pretty quickly when they appear.

Btw, we released v2.2.0 today. :)
